code/game/objects/items/weapons/cigs.dm

```diff
@@ -358,6 +359,71 @@ LIGHTERS ARE IN LIGHTERS.DM
 	else
 		to_chat(user, "<span class='notice'>[src] straight out REFUSES to be lit by such uncivilized means.</span>")
 
+/obj/item/clothing/mask/holo_cigar
+	name = "Holo-Cigar"
+	desc = "A sleek electronic cigar imported straight from Sol. You feel badass merely glimpsing it..."
+	icon_state = "holocigaroff"
+	item_state = "cigaroff"
```

Comment:
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/9a2d94a0-3f64-49d8-be6a-d433891d9b22)

Looks like you forgot to remove these, since you are now looking for "cigaroff" which has the same sprite.